File: examples/rss/remove/node/removeAFeed.js
import * as rss from "@m3o/services/rss";

// Remove an RSS feed by name
async function RemoveAfeed() {
  let rssService = new rss.RssService(process.env.MICRO_API_TOKEN);
  let rsp = await rssService.remove({
    name: "bbc",
  });
  console.log(rsp);
}

await RemoveAfeed();

File: examples/rss/remove/go/removeAFeed.go
package example

import (
	"fmt"
	"os"

	"github.com/micro/services/clients/go/rss"
)

// Remove an RSS feed by name
func RemoveAfeed() {
	rssService := rss.NewRssService(os.Getenv("MICRO_API_TOKEN"))
	rsp, err := rssService.Remove(&rss.RemoveRequest{
		Name: "bbc",
	})
	fmt.Println(rsp, err)
}
